Resources/Prototypes/Entities/Objects/Weapons/security.yml

```
@@ -44,11 +44,13 @@
     heavyStaminaCost: 1
     animation: WeaponArcSlash
   - type: DamageOtherOnHit
-  - type: StaminaDamageOnHit
-    damage: 35
+  - type: StaminaDamageOnHit # Stunmeta
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)